### PR TITLE
Display 'required' asterisk next to the 'Assignee' and 'Category' labels

### PR DESCRIPTION
**Pull request by tnewell5**
__Description of Change(s) Introduced:__ 
This PR displays an asterisk next to the required form labels: 'Assignee' and 'Category'.

__Description of Testing Applied:__ front-end test has been updated accordingly

__Relevant Screenshots/GIFs:__ 
image

image

__Relevant github Issue:__ 

- [issue (368)](https://github.com/stride-nyc/remote_retro/issues/368)

### CODE REVIEW

**Comment by vanderhoop** [comment]
hey @tnewell5, apologies for my delayed response. This should be scoped to the assignee field, per my response to your question  [here](https://github.com/stride-nyc/remote_retro/issues/368#issuecomment-359593854).

File: web/static/js/components/css_modules/idea_submission_form.css [hunk 1]
```diff
@@ -22,3 +22,7 @@
   top: -3em;
   left: 33%;
 }
+
+#required:after {
+  display: inline;
```
Comment (vanderhoop) [inline web/static/js/components/css_modules/idea_submission_form.css]: ahh, so that's all it took. i'm guessing the :after value was `display: block;`?

Comment (tnewell5) [reply web/static/js/components/css_modules/idea_submission_form.css]: it was 'display: inline-block;'

File: test/components/select_dropdown_test.js [hunk 2]
```diff
@@ -9,9 +9,25 @@ describe("SelectDropdown component", () => {
     value: "value",
     onChange() {},
     selectOptions: [],
+    isRequired: false,
   }
   let wrapper
 
+  context("when the stage is 'idea-generation'", () => {
```
Comment (vanderhoop) [inline test/components/select_dropdown_test.js]: need to update the test language here to reflect the new prop!

File: web/static/js/components/idea_edit_form.jsx [hunk 3]
```diff
@@ -78,13 +78,15 @@ class IdeaEditForm extends Component {
...
           showLabel={false}
+          isRequired={false}
         />}
         {stage === ACTION_ITEMS && <SelectDropdown
           labelName="editable_assignee"
           value={ideaAssigneeId || ""}
           onChange={this.onChangeAssignee}
           selectOptions={assigneeOptions}
           showLabel={false}
+          isRequired={false}
```
Comment (vanderhoop) [inline web/static/js/components/idea_edit_form.jsx]: I think here we'll want it to be `true`

Comment (vanderhoop) [reply web/static/js/components/idea_edit_form.jsx]: ahh! nevermind!

File: test/components/select_dropdown_test.js (hunk 2 above)
Comment (tnewell5) [reply test/components/select_dropdown_test.js]: cool - done!

**Pull request merged**